.travis.yml

```diff
@@ -21,4 +21,8 @@ script:
   - cabal build
   - cabal test
   - cabal install
+  - ([ "$GHCVER" = "7.6.3" ] || cabal configure --enable-tests --enable-executable-dynamic)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal build)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal test)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal install)
```

Comment:
Why does this need changing? It works currently on Travis. Or do the rest of the changes mean this is now required?